sdk/lusid/utilities/api_client_builder.py

```diff
@@ -13,62 +13,46 @@
 
 class ApiClientBuilder:
 
-    def build(self, api_configuration):
+    @staticmethod
+    def build(api_configuration, okta_response_handler = None):
```

Comment:
What is api_configuration? Is this a string which is the path to the secrets file? Be good to have its type in the function docstring i.e. :param (string) api_configuration

Reply:
Repeating comment from above. The ApiConfigurationLoader takes "api_configuration" as a parameter. As it returns the ApiConfiguration class I think this is a bit confusing. I'd suggest renaming this to something like "api_secrets_filename" or something to that effect and make it clear in the doc string of the load function of ApiConfigurationLoader that it is a string.